src/lexers/javascript-lexer.js

```diff
@@ -74,9 +74,17 @@ export default class JavascriptLexer extends BaseLexer {
         }
       }
 
+      if (this.defaultNamespace) {
+        entry.namespace = this.defaultNamespace;
```

Comment:
For consistency purposes, could you remove the `;`?

Reply:
sure, done